.pre-commit-config.yaml

```diff
@@ -0,0 +1,25 @@
+repos:
```

Comment:
So what does this check?

Reply:
This configures the pre-commit functionnality: when you prepare a commit, it will automatically run default pre-commit checks and run black and flake8 checks.